cannot load package "vcomponents" when building with Docker for Windows 10

Coming from https://github.com/antonblanchard/microwatt/issues/151#issuecomment-652855160, I'm trying to build microwatt on Windows 10, using Docker Desktop and MSYS2 (MINGW64). The procedure is to just clone the repo, change line 29 of the makefile as explained in https://github.com/antonblanchard/microwatt/pull/209#issuecomment-652031508, and run:

```sh
# DOCKER=1 make
docker run --rm -v //t/microwatt://src:z -w //src ghdl/ghdl:buster-llvm-7 ghdl -c --std=08 -frelaxed -Psim-unisim -Wl,sim_vhpi_c.o -Wl,sim_bram_helpers_c.o -Wl,sim_console_c.o -Wl,sim_jtag_socket_c.o decode_types.vhdl common.vhdl wishbone_types.vhdl fetch1.vhdl utils.vhdl 
plru.vhdl cache_ram.vhdl icache.vhdl decode1.vhdl helpers.vhdl insn_helpers.vhdl gpr_hazard.vhdl cr_hazard.vhdl control.vhdl decode2.vhdl register_file.vhdl cr_file.vhdl crhelpers.vhdl ppc_fx_insns.vhdl rotator.vhdl logical.vhdl countzero.vhdl multiply.vhdl divider.vhdl execute1.vhdl loadstore1.vhdl mmu.vhdl dcache.vhdl writeback.vhdl core_debug.vhdl core.vhdl wishbone_arbiter.vhdl wishbone_bram_wrapper.vhdl sync_fifo.vhdl wishbone_debug_master.vhdl xics.vhdl syscon.vhdl soc.vhdl spi_rxtx.vhdl spi_flash_ctrl.vhdl sim_console.vhdl sim_pp_uart.vhdl sim_bram_helpers.vhdl sim_bram.vhdl sim_jtag_socket.vhdl sim_jtag.vhdl dmi_dtm_xilinx.vhdl sim_16550_uart.vhdl core_tb.vhdl -e core_tb
error: cannot load package "vcomponents"
ghdl: compilation error
make: *** [Makefile:101: core_tb] Error 1
```

@ozbenh `sim-unisim/unisim-obj08.cf` exists, but it contains:

```
v 4
file "T:\microwatt\" "sim-unisim/unisim_vcomponents.vhdl" "917ee7a8dc19536a72a1b70ebd3a36b8d76af2d8" "20200702062346.912":
  package vcomponents at 1( 0) + 0 on 4;
file "T:\microwatt\" "sim-unisim/BUFG.vhdl" "6e19508c796a8fded15f2e3068675dbd302e9da1" "20200702062346.912":
  entity bufg at 1( 0) + 0 on 4;
  architecture behaviour of bufg at 9( 121) + 0 on 4;
file "T:\microwatt\" "sim-unisim/BSCANE2.vhdl" "05310f6fb5017fb11aa2bf46e8ae2db937570aa6" "20200702062346.912":
  entity bscane2 at 1( 0) + 0 on 4;
  architecture behaviour of bscane2 at 24( 469) + 0 on 4;
```

I guess that `"T:\microwatt\" "sim-unisim/unisim_vcomponents.vhdl"` should be `/t/microwatt/sim-unisim/unisim_vcomponents.vhdl` or just `/src/microwatt/sim-unisim/unisim_vcomponents.vhdl`. Can you please let me know the content in your host?

--- a/Makefile
+++ b/Makefile
@@ -71,7 +71,7 @@ unisim_lib = $(unisim_dir)/unisim-obj08.cf
 unisim_lib_files = $(unisim_dir)/BSCANE2.vhdl $(unisim_dir)/BUFG.vhdl \
 	$(unisim_dir)/unisim_vcomponents.vhdl
 $(unisim_lib): $(unisim_lib_files)
-	ghdl -i --std=08 --work=unisim --workdir=$(unisim_dir) $^
+	$(GHDL) -i --std=08 --work=unisim --workdir=$(unisim_dir) $^
 GHDLFLAGS += -P$(unisim_dir)
 
 core_tbs = multiply_tb divider_tb rotator_tb countzero_tb
@@ -85,7 +85,7 @@ fmf_lib = $(fmf_dir)/fmf-obj08.cf
 fmf_lib_files = $(wildcard $(fmf_dir)/*.vhd)
 GHDLFLAGS += -P$(fmf_dir)
 $(fmf_lib): $(fmf_lib_files)
-	ghdl -i --std=08 --work=fmf --workdir=$(fmf_dir) $^
+	$(GHDL) -i --std=08 --work=fmf --workdir=$(fmf_dir) $^
 
 flash_model_files=$(FLASH_MODEL_PATH)/s25fl128s.vhd
 flash_model_files: $(fmf_lib)

